src/interfaces/functions-in-interfaces.ts

```diff
@@ -0,0 +1,23 @@
+// Functions in Interfaces
+
+// Interfaces in TypeScript can define function signatures for clarity and consistency.
+
+// Example: Define an interface for a basic addition function of two numbers.
```

Comment:
we don't need to add `Example: bla bla ` since the code block is self explanatory and the consumer is the presentation (less is more)